dataflow-website/recipes/scaling/kubernetes/helm/alertwebhook/alertwebhook-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: alertwebhook
+  labels:
+    app: alertwebhook
+spec:
+  selector:
+    matchLabels:
+      app: alertwebhook
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: alertwebhook
+    spec:
+      containers:
+        - image: tzolov/scdf-alert-webhook:latest
```

Comment:
Whenever it is ready, we will need to swap it with the image from the samples Docker hub.